content/docs/studio/get-started.md

```diff
@@ -42,8 +42,9 @@ details]).
 
    </admon>
 
-3. Iterative Studio parses your project's `dvc.yaml` file to identify data,
-   metrics, plots and hyperparameters.
+3. Iterative Studio identifies experiments, which are Git commits or Git refs,
```

Comment:
Minor, non-blocking thought: Git refs (like branches and tags) are pointers to Git commits (so there is a commit for each dvc exp ref), so not sure we need to mention Git refs separately here.

Reply:
```suggestion
3. Iterative Studio identifies [experiments](doc/start/experiments), which are Git commits or Git refs,
```

Also not blocking, but I think it would be nice to link back to DVC docs at some points to hint where to get started for new users.

Reply:
For the above link, did you mean `/doc/start#experiment-management`? If yes, I've made the change in [this commit](https://github.com/iterative/dvc.org/pull/4491/commits/23f118d8f90718325572533090e478cfe79c219a).

Reply:
I meant to link to https://dvc.org/doc/start/experiments, but I don't think the difference is that important.